src/completions/History.ts

```diff
@@ -17,7 +19,6 @@ class HistoryCompletionOption extends Completions.CompletionOptionHTML
 
         // Create HTMLElement
         this.html = html`<tr class="HistoryCompletionOption option">
-            <td class="prefix">${"".padEnd(2)}</td>
```

Comment:
Was this doing anything?

Reply:
I found a use for it after all:
![image](https://user-images.githubusercontent.com/10012625/214625140-81b350be-1c6b-4a5f-a459-ca96c879235e.png)

With `completions.Tab.statusstylepretty=false`, I've marked bookmarks with `B` and searchurls with `S`.